.travis.yml

```diff
@@ -1,16 +1,13 @@
 language: python
-
-env:
-- CODECOV_TOKEN="d58091c4-cca2-4533-b419-bac51ed1511f"
-
 python:
-  - "2.7"
-
+  - "3.5"
 install:
-  - "pip install -r requirements.txt"
-
+  - pip install -r requirements_for_test.txt
```

Comment:
Yup, removing the _for_test version. This is a hangover from the auto-generator, Jon has optimised to incorporate the requirements.txt as a component of the test- version, I didn't spot the name change when I refreshed.